src/components/Home.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { loading } from '../features/country/countrySlice';
+import IconArrowRightCircle from './IconArrowRightCircle';
+
+const Home = () => {
+  const data = useSelector((state) => state.country);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(loading(false));
+  });
+
+  return (
+    <div className="home">
```

Comment:
- [ Required ] As suggested by the previous reviewer please ensure that on the homepage of your application, there is a hero section. A hero section is very important in an App because it contains information describing what the app is intended for and it also helps visitors to get a better understanding of what information the app offers. Please see the difference in the screenshot below and rectify this.

## Screenshots
| Your design | Behance design |
|-----------------|------------------------|
|![Screenshot from 2023-08-11 12-33-04](https://github.com/kerlos-bahaa/React-capstone/assets/16392046/42ac84d0-54c4-4d90-955e-bd3211905ad0)|![Screenshot from 2023-08-11 12-36-06](https://github.com/kerlos-bahaa/React-capstone/assets/16392046/33825dab-cba6-4280-be87-d0af81ff932f)|